Doc/library/importlib.rst

```diff
@@ -1241,6 +1241,74 @@ find and load modules.
     and how the module's :attr:`__file__` is populated.
 
 
+.. class:: AppleFrameworkLoader(fullname, dylib_path, parent_paths=None)
+
+   A specialization of :class:`importlib.machinery.ExtensionFileLoader` that
+   is able to load extension modules in Framework format.
+
+   For compatibility with the iOS App Store, *all* binary modules in an iOS app
+   must be ``.dylib objects``, contained in a framework with appropriate
+   metadata, stored in the ``Frameworks`` folder of the packaged app. There can
+   be only a single binary per framework, and there can be no executable binary
+   material outside the Frameworks folder.
+
+   If you're trying to run ``from foo.bar import _whiz``, and ``_whiz`` is
+   implemented with the binary module ``foo/bar/_whiz.abi3.dylib`` (or any
+   other ABI .dylib extension), this loader will look for
+   ``{sys.executable}/Frameworks/foo.bar._whiz.framework/_whiz.abi3.dylib``
+   (forming the package name by taking the full import path of the library,
+   and replacing ``/`` with ``.``).
```

Comment:
```suggestion
   ``{dirname(sys.executable)}/Frameworks/foo.bar._whiz.framework/_whiz.abi3.dylib``
   (forming the framework name from the full name of the module).
```

Reply:
Also, it isn't the loader that has this functionality, but the finder.

Reply:
Good point; I'll rework these docs to be associated with the finder, not the loader, and adjust the discussion of the ``__file__`` attribute accordingly.